extra/release_tool.py

```diff
@@ -172,6 +172,7 @@ def is_release_component(self):
     "deviceauth": "DEVICEAUTH_REV",
     "gui": "GUI_REV",
     "inventory": "INVENTORY_REV",
+    "inventory-enterprise": "INVENTORY_ENTERPRISE_REV",
```

Comment:
ping @tranchitella: to be added to the list of things to do on new repos.

We could think about a better way to do this, though. :thinking:

Reply:
There has been quite a few pieces needed for this one simple thing...